test/parallel/test-fs-access.js

```diff
@@ -87,11 +87,11 @@ assert.throws(() => {
 
 assert.throws(() => {
   fs.access(__filename, fs.F_OK);
-}, /^TypeError: "callback" argument must be a function$/);
+}, /^TypeError \[ERR_INVALID_CALLBACK\]: callback must be a function$/);
```

Comment:
These can be changed to use `common.expectsError({code: 'ERR_INVALID_CALLBACK'})`